rails/app/views/layouts/minimal.html.haml

```diff
@@ -30,6 +30,8 @@
     %script
       try{Typekit.load({ async: true });}catch(e){}
 
+    %script{:src => "https://kit.fontawesome.com/32aedade6e.js", :crossorigin => "anonymous"}
```

Comment:
@scytacki We don't have to do that now, but probably should update the account settings so we do. What should we add to a list of authorized domains?

*.concord.org
*.concordqa.org
app.portal.docker (not sure this would work)